File: src/app/gkill/api/req_res/update_ur_log_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

type UpdateURLogResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	UpdatedURLog *reps.URLog `json:"updated_urlog"`

	UpdatedURLogKyou *reps.Kyou `json:"updated_urlog_kyou"`
}

File: src/classes/api/req_res/update-ur-log-response.ts
'use strict';

import { URLog } from '@/classes/datas/ur-log';
import { GkillAPIResponse } from '../gkill-api-response';
import { Kyou } from '@/classes/datas/kyou';


export class UpdateURLogResponse extends GkillAPIResponse {


    updated_urlog: URLog;

    updated_urlog_kyou: Kyou;

    constructor() {
        super()
        this.updated_urlog = new URLog()
        this.updated_urlog_kyou = new Kyou()
    }


}



